drivers/gles3/storage/render_scene_buffers_gles3.cpp

```diff
@@ -50,10 +60,17 @@ void RenderSceneBuffersGLES3::configure(RID p_render_target, const Size2i p_inte
 	//msaa = p_msaa;
 	//screen_space_aa = p_screen_space_aa;
 	//use_debanding = p_use_debanding;
-	//view_count = p_view_count;
+	view_count = p_view_count;
 
 	free_render_buffer_data();
 
+#if !defined(GLES_OVER_GL) && !defined(WEB_ENABLED)
+	PFNGLFRAMEBUFFERTEXTUREMULTIVIEWOVRPROC glFramebufferTextureMultiviewOVR = nullptr;
+	if (config->multiview_supported && view_count > 1) {
+		glFramebufferTextureMultiviewOVR = (PFNGLFRAMEBUFFERTEXTUREMULTIVIEWOVRPROC)eglGetProcAddress("glFramebufferTextureMultiviewOVR");
+	}
+#endif
+
```

Comment:
This should move to either Config or RasterizerGLES3 so that it isn't initialized every time it is used and can be shared

Reply:
Sure! My thinking was that this doesn't happen very often (only when the render targets are resized and there is more than one view, which is pretty much just when first entering XR mode), so I thought I could get away with it :-)

Reply:
Yeah have a look at how we do this in OpenGL itself where we need to grab these pointers, pointers are obtained at init, then used with a check whether a pointer was obtained (remember these can fail if something is not supported!)

Reply:
For the most part, the function pointers are loaded via GLAD. That's actually the case here too on the desktop, I think it's only Android where the above code is actually needed